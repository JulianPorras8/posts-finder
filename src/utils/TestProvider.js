/* eslint-disable react/prop-types */
import React from 'react';
import { Provider } from 'react-redux';
import { ThemeProvider } from '@material-ui/styles';

import configureStore from 'redux-mock-store';
import theme from '../theme';

const mockStore = configureStore([]);

const initialState = {
  posts: {
    items: [{
      id: 1,
      userId: 1,
      title: 'test 1',
      body: 'post Body 1',
    }, {
      id: 2,
      userId: 1,
      title: 'test 2',
      body: 'post Body 2',
    }],
    isLoading: false,
    error: '',
    pageNumber: 1,
    posts: [],
    preSelectedPost: null,
    selectedPost: null,
  }
};
const store = mockStore(initialState);

export default function TestProvider({ children }) {
  return (
    <ThemeProvider theme={theme}>
      <Provider store={store}>
        {children}
      </Provider>
    </ThemeProvider>
  );
}
