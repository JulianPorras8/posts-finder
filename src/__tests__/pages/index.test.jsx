// import 'jsdom-global/register';
import React from 'react';
import { render } from '@testing-library/react';

import TestProvider from '../../utils/TestProvider';

// jest.mock('redux-persist/lib/storage', () => {
//   var store = {};
//   return {
//     default: {
//       getItem: function (key) {
//         return store[key];
//       },
//       setItem: function (key, value) {
//         store[key] = value.toString();
//       },
//       clear: function () {
//         store = {};
//       },
//       removeItem: function (key) {
//         delete store[key];
//       }
//     }
//   };
// });

import Index from '../../pages/index';

describe('<Index />', () => {

  const app = render(
    <TestProvider>
      <Index />
    </TestProvider>
  );

  it('matches snapshot', () => {
    expect(app).toMatchSnapshot();
  });

  describe('<RegisterContainer />', () => {
    it('renders todo input', () => {
      expect(app.container).toBeTruthy();
    });
  });
});
