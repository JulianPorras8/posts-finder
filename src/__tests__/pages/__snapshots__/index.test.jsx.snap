// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="makeStyles-root-1"
      >
        <div
          class="makeStyles-appFrame-2"
        >
          <header
            class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-appBar-3 mui-fixed MuiPaper-elevation4"
          >
            <div
              class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit"
              >
                Avantica - Posts Finder
              </h6>
            </div>
          </header>
          <div
            class="makeStyles-content-5"
          >
            <div
              class="MuiGrid-root makeStyles-pageContent-6 MuiGrid-container"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                    >
                      <div
                        aria-expanded="false"
                        class="MuiAutocomplete-root makeStyles-searchControl-11 MuiAutocomplete-hasClearIcon MuiAutocomplete-hasPopupIcon"
                        role="combobox"
                      >
                        <div
                          class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                        >
                          <label
                            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                            data-shrink="false"
                            for="asynchronous-issues"
                            id="asynchronous-issues-label"
                          >
                            Post
                          </label>
                          <div
                            class="MuiInputBase-root MuiOutlinedInput-root MuiAutocomplete-inputRoot MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
                          >
                            <input
                              aria-autocomplete="both"
                              aria-invalid="false"
                              autocapitalize="none"
                              autocomplete="off"
                              class="MuiInputBase-input MuiOutlinedInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
                              id="asynchronous-issues"
                              spellcheck="false"
                              type="text"
                              value=""
                            />
                            <div
                              class="MuiAutocomplete-endAdornment"
                            >
                              <button
                                aria-label="Clear"
                                class="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-clearIndicator"
                                tabindex="-1"
                                title="Clear"
                                type="button"
                              >
                                <span
                                  class="MuiIconButton-label"
                                >
                                  <svg
                                    aria-hidden="true"
                                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                                    focusable="false"
                                    viewBox="0 0 24 24"
                                  >
                                    <path
                                      d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  class="MuiTouchRipple-root"
                                />
                              </button>
                              <button
                                aria-label="Open"
                                class="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-popupIndicator"
                                tabindex="-1"
                                title="Open"
                                type="button"
                              >
                                <span
                                  class="MuiIconButton-label"
                                >
                                  <svg
                                    aria-hidden="true"
                                    class="MuiSvgIcon-root"
                                    focusable="false"
                                    viewBox="0 0 24 24"
                                  >
                                    <path
                                      d="M7 10l5 5 5-5z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  class="MuiTouchRipple-root"
                                />
                              </button>
                            </div>
                            <fieldset
                              aria-hidden="true"
                              class="PrivateNotchedOutline-root-12 MuiOutlinedInput-notchedOutline"
                            >
                              <legend
                                class="PrivateNotchedOutline-legendLabelled-14"
                              >
                                <span>
                                  Post
                                </span>
                              </legend>
                            </fieldset>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                    >
                      <ul
                        class="MuiList-root makeStyles-list-16 MuiList-padding"
                      >
                        <div>
                          <li
                            class="MuiListItem-container"
                          >
                            <div
                              class="MuiListItem-root MuiListItem-gutters MuiListItem-alignItemsFlexStart MuiListItem-secondaryAction"
                            >
                              <div
                                class="MuiListItemAvatar-root MuiListItemAvatar-alignItemsFlexStart"
                              >
                                <div
                                  class="MuiAvatar-root MuiAvatar-circle MuiAvatar-colorDefault"
                                >
                                  <svg
                                    aria-hidden="true"
                                    class="MuiSvgIcon-root"
                                    focusable="false"
                                    viewBox="0 0 24 24"
                                  >
                                    <path
                                      d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                                    />
                                  </svg>
                                </div>
                              </div>
                              <div
                                class="MuiListItemText-root MuiListItemText-multiline"
                              >
                                <span
                                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                                >
                                  test 1
                                </span>
                                <p
                                  class="MuiTypography-root MuiListItemText-secondary MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
                                >
                                  post Body 1
                                </p>
                              </div>
                            </div>
                            <div
                              class="MuiListItemSecondaryAction-root"
                            >
                              <button
                                aria-label="delete"
                                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd"
                                tabindex="0"
                                type="button"
                              >
                                <span
                                  class="MuiIconButton-label"
                                >
                                  <svg
                                    aria-hidden="true"
                                    class="MuiSvgIcon-root"
                                    focusable="false"
                                    viewBox="0 0 24 24"
                                  >
                                    <path
                                      d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  class="MuiTouchRipple-root"
                                />
                              </button>
                            </div>
                          </li>
                          <li
                            class="MuiDivider-root"
                            role="separator"
                          />
                        </div>
                        <div>
                          <li
                            class="MuiListItem-container"
                          >
                            <div
                              class="MuiListItem-root MuiListItem-gutters MuiListItem-alignItemsFlexStart MuiListItem-secondaryAction"
                            >
                              <div
                                class="MuiListItemAvatar-root MuiListItemAvatar-alignItemsFlexStart"
                              >
                                <div
                                  class="MuiAvatar-root MuiAvatar-circle MuiAvatar-colorDefault"
                                >
                                  <svg
                                    aria-hidden="true"
                                    class="MuiSvgIcon-root"
                                    focusable="false"
                                    viewBox="0 0 24 24"
                                  >
                                    <path
                                      d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                                    />
                                  </svg>
                                </div>
                              </div>
                              <div
                                class="MuiListItemText-root MuiListItemText-multiline"
                              >
                                <span
                                  class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                                >
                                  test 2
                                </span>
                                <p
                                  class="MuiTypography-root MuiListItemText-secondary MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
                                >
                                  post Body 2
                                </p>
                              </div>
                            </div>
                            <div
                              class="MuiListItemSecondaryAction-root"
                            >
                              <button
                                aria-label="delete"
                                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd"
                                tabindex="0"
                                type="button"
                              >
                                <span
                                  class="MuiIconButton-label"
                                >
                                  <svg
                                    aria-hidden="true"
                                    class="MuiSvgIcon-root"
                                    focusable="false"
                                    viewBox="0 0 24 24"
                                  >
                                    <path
                                      d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"
                                    />
                                  </svg>
                                </span>
                                <span
                                  class="MuiTouchRipple-root"
                                />
                              </button>
                            </div>
                          </li>
                          <li
                            class="MuiDivider-root"
                            role="separator"
                          />
                        </div>
                      </ul>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-sm-12 MuiGrid-grid-md-8"
                    >
                      <nav
                        aria-label="pagination navigation"
                        class="MuiPagination-root"
                      >
                        <ul
                          class="MuiPagination-ul"
                        >
                          <li>
                            <button
                              aria-label="Go to previous page"
                              class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-textPrimary Mui-disabled Mui-disabled"
                              disabled=""
                              tabindex="-1"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiPaginationItem-icon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                                />
                              </svg>
                            </button>
                          </li>
                          <li>
                            <button
                              aria-current="true"
                              aria-label="page 1"
                              class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-textPrimary Mui-selected"
                              tabindex="0"
                              type="button"
                            >
                              1
                              <span
                                class="MuiTouchRipple-root"
                              />
                            </button>
                          </li>
                          <li>
                            <button
                              aria-label="Go to next page"
                              class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-textPrimary Mui-disabled Mui-disabled"
                              disabled=""
                              tabindex="-1"
                              type="button"
                            >
                              <svg
                                aria-hidden="true"
                                class="MuiSvgIcon-root MuiPaginationItem-icon"
                                focusable="false"
                                viewBox="0 0 24 24"
                              >
                                <path
                                  d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                                />
                              </svg>
                            </button>
                          </li>
                        </ul>
                      </nav>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="makeStyles-root-1"
    >
      <div
        class="makeStyles-appFrame-2"
      >
        <header
          class="MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-appBar-3 mui-fixed MuiPaper-elevation4"
        >
          <div
            class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit"
            >
              Avantica - Posts Finder
            </h6>
          </div>
        </header>
        <div
          class="makeStyles-content-5"
        >
          <div
            class="MuiGrid-root makeStyles-pageContent-6 MuiGrid-container"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
                  >
                    <div
                      aria-expanded="false"
                      class="MuiAutocomplete-root makeStyles-searchControl-11 MuiAutocomplete-hasClearIcon MuiAutocomplete-hasPopupIcon"
                      role="combobox"
                    >
                      <div
                        class="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                      >
                        <label
                          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                          data-shrink="false"
                          for="asynchronous-issues"
                          id="asynchronous-issues-label"
                        >
                          Post
                        </label>
                        <div
                          class="MuiInputBase-root MuiOutlinedInput-root MuiAutocomplete-inputRoot MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
                        >
                          <input
                            aria-autocomplete="both"
                            aria-invalid="false"
                            autocapitalize="none"
                            autocomplete="off"
                            class="MuiInputBase-input MuiOutlinedInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
                            id="asynchronous-issues"
                            spellcheck="false"
                            type="text"
                            value=""
                          />
                          <div
                            class="MuiAutocomplete-endAdornment"
                          >
                            <button
                              aria-label="Clear"
                              class="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-clearIndicator"
                              tabindex="-1"
                              title="Clear"
                              type="button"
                            >
                              <span
                                class="MuiIconButton-label"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                                  />
                                </svg>
                              </span>
                              <span
                                class="MuiTouchRipple-root"
                              />
                            </button>
                            <button
                              aria-label="Open"
                              class="MuiButtonBase-root MuiIconButton-root MuiAutocomplete-popupIndicator"
                              tabindex="-1"
                              title="Open"
                              type="button"
                            >
                              <span
                                class="MuiIconButton-label"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M7 10l5 5 5-5z"
                                  />
                                </svg>
                              </span>
                              <span
                                class="MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                          <fieldset
                            aria-hidden="true"
                            class="PrivateNotchedOutline-root-12 MuiOutlinedInput-notchedOutline"
                          >
                            <legend
                              class="PrivateNotchedOutline-legendLabelled-14"
                            >
                              <span>
                                Post
                              </span>
                            </legend>
                          </fieldset>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
                  >
                    <ul
                      class="MuiList-root makeStyles-list-16 MuiList-padding"
                    >
                      <div>
                        <li
                          class="MuiListItem-container"
                        >
                          <div
                            class="MuiListItem-root MuiListItem-gutters MuiListItem-alignItemsFlexStart MuiListItem-secondaryAction"
                          >
                            <div
                              class="MuiListItemAvatar-root MuiListItemAvatar-alignItemsFlexStart"
                            >
                              <div
                                class="MuiAvatar-root MuiAvatar-circle MuiAvatar-colorDefault"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                                  />
                                </svg>
                              </div>
                            </div>
                            <div
                              class="MuiListItemText-root MuiListItemText-multiline"
                            >
                              <span
                                class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                              >
                                test 1
                              </span>
                              <p
                                class="MuiTypography-root MuiListItemText-secondary MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
                              >
                                post Body 1
                              </p>
                            </div>
                          </div>
                          <div
                            class="MuiListItemSecondaryAction-root"
                          >
                            <button
                              aria-label="delete"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd"
                              tabindex="0"
                              type="button"
                            >
                              <span
                                class="MuiIconButton-label"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"
                                  />
                                </svg>
                              </span>
                              <span
                                class="MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </li>
                        <li
                          class="MuiDivider-root"
                          role="separator"
                        />
                      </div>
                      <div>
                        <li
                          class="MuiListItem-container"
                        >
                          <div
                            class="MuiListItem-root MuiListItem-gutters MuiListItem-alignItemsFlexStart MuiListItem-secondaryAction"
                          >
                            <div
                              class="MuiListItemAvatar-root MuiListItemAvatar-alignItemsFlexStart"
                            >
                              <div
                                class="MuiAvatar-root MuiAvatar-circle MuiAvatar-colorDefault"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                                  />
                                </svg>
                              </div>
                            </div>
                            <div
                              class="MuiListItemText-root MuiListItemText-multiline"
                            >
                              <span
                                class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                              >
                                test 2
                              </span>
                              <p
                                class="MuiTypography-root MuiListItemText-secondary MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-displayBlock"
                              >
                                post Body 2
                              </p>
                            </div>
                          </div>
                          <div
                            class="MuiListItemSecondaryAction-root"
                          >
                            <button
                              aria-label="delete"
                              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd"
                              tabindex="0"
                              type="button"
                            >
                              <span
                                class="MuiIconButton-label"
                              >
                                <svg
                                  aria-hidden="true"
                                  class="MuiSvgIcon-root"
                                  focusable="false"
                                  viewBox="0 0 24 24"
                                >
                                  <path
                                    d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5zM12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"
                                  />
                                </svg>
                              </span>
                              <span
                                class="MuiTouchRipple-root"
                              />
                            </button>
                          </div>
                        </li>
                        <li
                          class="MuiDivider-root"
                          role="separator"
                        />
                      </div>
                    </ul>
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-sm-12 MuiGrid-grid-md-8"
                  >
                    <nav
                      aria-label="pagination navigation"
                      class="MuiPagination-root"
                    >
                      <ul
                        class="MuiPagination-ul"
                      >
                        <li>
                          <button
                            aria-label="Go to previous page"
                            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-textPrimary Mui-disabled Mui-disabled"
                            disabled=""
                            tabindex="-1"
                            type="button"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiPaginationItem-icon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                              />
                            </svg>
                          </button>
                        </li>
                        <li>
                          <button
                            aria-current="true"
                            aria-label="page 1"
                            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-textPrimary Mui-selected"
                            tabindex="0"
                            type="button"
                          >
                            1
                            <span
                              class="MuiTouchRipple-root"
                            />
                          </button>
                        </li>
                        <li>
                          <button
                            aria-label="Go to next page"
                            class="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-page MuiPaginationItem-textPrimary Mui-disabled Mui-disabled"
                            disabled=""
                            tabindex="-1"
                            type="button"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiPaginationItem-icon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                              />
                            </svg>
                          </button>
                        </li>
                      </ul>
                    </nav>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
